src/main/java/cz/cvut/kbss/termit/persistence/dao/UserRoleDao.java

```diff
@@ -35,4 +36,9 @@ public UserRoleDao(EntityManager em) {
     public List<UserRole> findAll() {
         return em.createQuery("SELECT r FROM " + UserRole.class.getSimpleName() + " r", UserRole.class).getResultList();
     }
+
+    public UserRole find(cz.cvut.kbss.termit.security.model.UserRole userRole) {
```

Comment:
Just em `em.find`, no query is needed.

Reply:
In addition, to be consistent with `find` in other DAOs, I suggest returning an `Optional` and handling missing value in the service.